packages/code-du-travail-frontend/src/common/Answer.js

```diff
@@ -159,14 +134,50 @@ const DefinitonTerm = ({ term, definition }) => {
   );
 };
 
-const { box, colors, fonts, spacing } = theme;
+const { box, breakpoints, colors, fonts, spacing } = theme;
 
 const StyledContainer = styled(Container)`
   margin: 20%;
   font-size: ${fonts.sizeH2};
   text-align: center;
 `;
 
+const StyledWrapper = styled(Container)`
+  display: flex;
+  flex-wrap: wrap;
+  padding: 0;
+`;
+
+const StyledContent = styled.div`
+  width: 70%;
+  @media (max-width: ${breakpoints.tablet}) {
+    width: 100%;
+  }
+`;
+
+const StyledMenu = styled.div`
+  padding: 2rem 0;
+  width: 30%;
+  color: ${colors.blue};
+  & a {
+    text-decoration: none;
+  }
+  @media (max-width: ${breakpoints.tablet}) {
+    display: none;
+  }
+`;
+
+const StyledMenuList = styled.div`
+  position: -webkit-sticky;
```

Comment:
styled-component should handle this for us. it is not he  case for sticky ?

Reply:
Done